src/v3/src/components/Widget/index.tsx

```diff
@@ -278,27 +285,29 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
     }}
     >
       {/* Note that we need two theme providers until we fully migrate to odyssey-mui */}
```

Comment:
Let's update this comment now, since we need to long-term keep both providers, one for MUI, and one for OdysseyMUI